### DISCUSSION TITLE
Displaying Entered Text in AR with Model Viewer?

### DISCUSSION THREAD

**Discussion by abhirami100** [google/model-viewer#4751] [Q&A]
Is there a feature in the model viewer that allows inputting text and then displaying that text in augmented reality?

**Comment by elalish** [comment]
You can use any standard HTML you want - however HTML is only available in browser-based AR, meaning WebXR, which is only supported on Android with Chrome-based browsers, as far as I know. See https://modelviewer.dev/examples/augmentedreality/

**Reply by abhirami100** [reply]
Could you provide an example of integrating text input onto a 3D model in an augmented reality  environment?

**Reply by elalish** [reply]
I gave you one (technically it puts images instead of text, but surely you can see the concept?). There are also many others (most of our examples can be used in AR). Please try them all out on an Android device in Chrome and see which ones are closest to what you want. Use the menu button in the upper-left to navigate all the examples.
